analyzer/src/funTest/kotlin/managers/Extensions.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.analyzer.managers
+
+import io.kotest.matchers.shouldNotBe
+
+import java.io.File
+
+import org.ossreviewtoolkit.analyzer.PackageManager
+import org.ossreviewtoolkit.model.ProjectAnalyzerResult
+
+fun PackageManager.resolveSingleProject(definitionFile: File): ProjectAnalyzerResult =
```

Comment:
Consider overloading the function instead, e.g. naming it `resolveDependencies()` or `resolveProjectDependencies()` alternatively.

Reply:
I agree having this overload as part of the public API could make sense.

Reply:
Adding this to the public API would conflict with another change I'm working on to allow multiple projects for a single definition file. Having this helper function in the tests makes it easier to implement that without having to change all package manager tests.